docs/deployment/containers/docker.rst

```diff
@@ -88,31 +88,31 @@ Breaking the command down:
   to allocate 2G for its heap
 - Runs the command ``crate`` inside the container with parameters:
     * ``network.host``: The ``_site_`` value results in the binding of the
```

Comment:
I think it is better to do lowercase after a colon throughout, unless it would really be confusing to do so.